NEWS

```diff
@@ -124,10 +123,11 @@
  * Fixed GDR copy performance regression by providing maximum estimation between get and put
  * Fixed CUDA IPC reachability check
  * Fixed crash in MPI_Finalize when CUDA context is destroyed
- * Always require rcache by default for GDR
- * Fixed md_close when no rcache
+ * Always require rcache by default for GDR copy
+ * Fixed crash GDR copy in UCT MD close when rcache was not enabled
```

Comment:
Fixed crash in gdr_copy cleanup when registration cache is disabled

Reply:
fixed